assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -73,8 +73,11 @@ def run_service(self):
                             continue
                         else:
                             raise
+                try:
+                    self.process_cluster(cluster)
+                except Exception as e:
+                    log.warning(f'Error while processing cluster {cluster_id}: {e}')
```

Comment:
maybe 
```suggestion
                    log.exception(f'Error while processing cluster {cluster_id}')
```

so that we can have a better traceback around those kind of errors